src/lib/Libtpp/tpp_router.c

```diff
@@ -1949,10 +1949,19 @@ router_pkt_handler(int tfd, void *data, int len, void *c, void *extra)
 		case TPP_MCAST_DATA: {
 			int i, k;
 			tpp_addr_t *src_host;
-			int *rlist = NULL;
+			typedef struct {
+				char *router_name;
+				int target_fd; /* target comm fd */
+				void *cmpr_ctx;
+				void *minfo_buf; /* allocate size for total members */
+				int num_streams; /* actual number of destination streams */ 
```

Comment:
shall we have int(s) after all the pointers or vice versa?

Reply:
if it makes it look slightly nicer, sure